pkgs/applications/networking/seafdav/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, python3
+, makeWrapper
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "seafdav";
+  version = "8.0.8";
```

Comment:
seafile-server has been updated to 9.0.x, maybe update this component too?

Reply:
Yes now that the other packages have been updated this should be too. I pushed the `9.0.7` version.